Formula/box2d.rb

```diff
@@ -1,11 +1,23 @@
 class Box2d < Formula
   desc "2D physics engine for games"
   homepage "https://box2d.org"
-  url "https://github.com/erincatto/box2d/archive/v2.4.1.tar.gz"
-  sha256 "d6b4650ff897ee1ead27cf77a5933ea197cbeef6705638dd181adc2e816b23c2"
   license "MIT"
+  revision 1
   head "https://github.com/erincatto/Box2D.git", branch: "main"
 
+  stable do
+    url "https://github.com/erincatto/box2d/archive/v2.4.1.tar.gz"
+    sha256 "d6b4650ff897ee1ead27cf77a5933ea197cbeef6705638dd181adc2e816b23c2"
+
+    # Update doctest to run on ARM. Remove in the next release.
+    patch do
+      on_arm do
+        url "https://github.com/erincatto/box2d/commit/cd2c28dba83e4f359d08aeb7b70afd9e35e39eda.patch?full_index=1"
+        sha256 "7636039eed6df7c32659870eb46a04b92c5dac827aeb23aadfb3ac51cb586b40"
+      end
+    end
```

Comment:
Is this still needed if we're using the `doctest` formula?

Reply:
No probably not, I'll drop this and see if it works without it.

Reply:
I dropped the patch and it works on ARM.  So the ARM fix is now just to use brewed `doctest` instead of the vendored copy.